docs/source/workflow_outputs.rst

```diff
@@ -162,29 +162,26 @@ Current annual building loads are listed below.
    Load: Hot Water: Solar Thermal (MBtu) Load served by the solar thermal system.
    ===================================== ==================================================================
 
-Annual Unmet Building Loads
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Note that heating and cooling loads represent the delivered energy by the HVAC system; if the HVAC system is significantly undersized, there will be unmet load not reflected by these values.
```

Comment:
I noticed that heating and cooling peak loads are also reduced due to the unmet loads. I haven't kept track of how peak loads are calculated and reported, but they also seem to be related to the hvac delivered energy instead of load calculation.

Reply:
Thanks for looking over this PR in such detail!

Good point, I will update the documentation for Peak Loads too. This also makes me think that it's probably a good idea to rename these outputs from, e.g., "Load: Heating" to "Load: Heating: Delivered". We already have a "Load: Hot Water: Delivered". (This reminds me that we should also have some language in the documentation about unmet hot water loads.) I hate to make breaking changes to output names, but it seems important. I will make these changes.